sound/soc/sof/ipc.c

```diff
@@ -312,10 +312,8 @@ static struct snd_sof_ipc_msg *sof_ipc_reply_find_msg(struct snd_sof_ipc *ipc,
 	struct snd_sof_dev *sdev = ipc->sdev;
 	struct snd_sof_ipc_msg *msg;
 
-	header = SOF_IPC_MESSAGE_ID(header);
-
 	list_for_each_entry(msg, &ipc->reply_list, list) {
-		if (SOF_IPC_MESSAGE_ID(msg->header) == header)
```

Comment:
@keyonjie could you explain where those low 16 bits of IPC commands, masked by SOF_IPC_MESSAGE_ID(), are actually set? I've searched through both the driver and the firmware IPC code and see a few locations, where those bits are read and checked, but I don't see them anywhere being set?

Reply:
@lyakh they are actually not set(0s low 16 bits for all IPCs), so that's the problem I reported here: https://github.com/thesofproject/linux/issues/516

Here my RFC PR try to remove comparing of this message ID(it is a wrong usage as all comparing were success here actually).